src/v/serde/serde.h

```diff
@@ -9,21 +9,21 @@
 
 #pragma once
 
-#include "serde/async.h"
```

Comment:
As far as I understand we generally encourage include paths relative to `srv/v`. Maybe @dotnwat or @BenPope can correct me here.

Can we consider reverting cd7af1c2d829eb7a485839dd7fec09bf678f7333 if that's the case?

Reply:
I removed the commit. It somehow fixed a compilation problem back then, but doesn't seem to be necessary anymore.